guides/common/modules/proc_examining-compliance-failures-of-a-host.adoc

```diff
@@ -0,0 +1,34 @@
+[id="Examining_Compliance_Failures_of_a_Host_{context}"]
+= Examining Compliance Failures of a Host
+
+Use this procedure to examine a full compliance report, determine why a host failed compliance on a rule, and in some cases, find out how to bring the host into compliance using a remediation.
```

Comment:
```suggestion
Using this procedure, a full compliance report can be examined to determine why a host failed compliance on a rule, and in some cases, how to remediate the issue.
```
Another option if you wish.

Reply:
@mjivraja Thank you, but we shouldn't use passive voice.